Public/Src/FrontEnd/Script/Ast/Expressions/SwitchExpressionClause.cs

```diff
@@ -36,18 +39,40 @@ public class SwitchExpressionClause : Expression
             Expression = expression;
         }
 
+        /// <nodoc />
+        public SwitchExpressionClause(

```

Comment:
>SwitchExpressionClause [](start = 15, length = 22)

maybe explicitly passing IsDefaultFallThrough on construction? 
I find 'true' being the default slightly misleading. And the pattern of creating the clause and immediately setting 'false' for the other non-default case sort not super clean

Reply:
Wouldn't that be weird, since it would have to fail on false... since false implies there must be a match expression

---
In reply to: [274188741](https://github.com/Microsoft/BuildXL/pull/123#discussion_r274188741) [](ancestors = 274188741)

Reply:
Oh, sorry, didn't notice there was another constructor with the match as a parameter. I think what set my off was that on the parser you are doing

node.IsDefaultFallthrough = match == null;

after constructing it, for every case

---
In reply to: [274201164](https://github.com/Microsoft/BuildXL/pull/123#discussion_r274201164) [](ancestors = 274201164,274188741)